moha/rauk/rauk.py

```diff
@@ -0,0 +1,175 @@
+import numpy as np 
+
+from scipy import constants 
+
+from scipy.special import gamma
+
+from moha.utils import get_atom_type
+
+from scipy.sparse import csr_matrix, diags
+import os
+import json
+
+def build_one_body_matrix(connectivity, atom_dictionary, atoms_num, n_sites, bond_dictionary):
```

Comment:
this function shouldn't need n_sites, atoms_num and atom_dictionary. 
1. n_sites can be parsed directly from the connectivity
2. atoms_num can be also parsed from the connectivity
3. atom_dictionary can be loaded from json file